src/doc/CMakeLists.txt

```diff
@@ -19,27 +19,30 @@
 #   same for all platforms, except the sphinx build command which is
 #   different on Windows.  Added logic to ensure sphinx build command exists.
 #
+#   Eric Brugger, Wed Feb 10 15:07:41 PST 2021
+#   Update for switch to Python 3.
+#
 #****************************************************************************
 
-if(VISIT_PYTHON3_DIR AND VISIT_ENABLE_MANUALS)
+if(VISIT_ENABLE_MANUALS)
```

Comment:
Since we use VISIT_PYTHON_DIR in the code below, I think the change should be 
if(VISIT_PYTHON_DIR and VISIT_ENABLE_MANUALS) ...
Sorry for my comment in the bug ticket suggesting removal would be okay.

Reply:
Changed.